macros/sfdc_formula_view.sql

```diff
@@ -33,7 +33,9 @@
 -- defaults to all formula fields if fields_to_include is none
 {% if fields_to_include is none %}
     {% set fields_to_include = current_formula_fields | lower %}
-{% endif %}
+{% endif %} 
+
+{{ exceptions.warn("WARNING, Deprecated: full_statement_version=false is no longer supported and will stop working on October 1st, 2023. See https://github.com/fivetran/dbt_salesforce_formula_utils#step-4-create-models for more details.") }}
```

Comment:
When looking at the logs it is hard to decipher where this message is coming from and where to act on this. Since customers can use a combination of `false` and `true` configs, I feel it would be valuable to add a more explicit warning. I also wanted to add breaks in the warnings as they all just clumped together in the logs without.

![image](https://github.com/fivetran/dbt_salesforce_formula_utils/assets/74217849/3c3fbf6c-793a-47d0-b819-018d9c1279e9)


```suggestion
{{ exceptions.warn("\nWARNING, Deprecated: full_statement_version=false is no longer supported and will stop working on October 1st, 2023. Be sure to update your " ~ this.identifier|upper ~ " model to leverage full_statement_version=true before then. See https://github.com/fivetran/dbt_salesforce_formula_utils#step-4-create-models for more details.\n") }}
```